### ISSUE TITLE
Can't find app with identifier com.themacronaut.net.phere.GitX

### ISSUE THREAD

**Issue by jerodsanto** [gitx/gitx#344]
I have GitX `0.23-beta2.2877` in my Applications directory and have enabled terminal usage. I can run `gitx -h` without problem, but when I run `gitx` without any arguments I get this error:

```
Can't find app with identifier com.themacronaut.net.phere.GitX
```

Happy to provide more information if it'll help. 

_(I'm SO excited y'all resurrected this project! It's my fav non-CLI git tool by far 💚)_

**Comment by hannesa2** [comment]
The changes from `0.23-beta2` were never merged, please use `0.23`.
I expect a working system with it.

**Comment by jerodsanto** [comment]
Good call, you are absolutely right!
